### PR TITLE
Make about error pages scrollable

### LINKED ISSUE #2152: about error pages are unscrollable
**Describe the issue you encountered:** about error pages are unscrollable, which might be annoying for users with a small screen computer.

**Expected behavior:** They should be scrollable for small screens.

I'm pushing a PR to fix it.

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

Closes #2152

This commit makes the about error pages, about:certerror, about:error and about:safebrowsing, scrollable.

Overflow of the retry button will be fixed by merging #2149

### CODE REVIEW

File: js/about/safebrowsing.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ class SafebrowsingPage extends React.Component {
   }
 
   render () {
-    return <div className='errorContent' id='safebrowsingPage' style={{height: 'auto'}}>
```
Comment (luixxiul) [inline js/about/safebrowsing.js]: I was not sure what was the style for. If requred for some reason I will revert this.

File: less/about/error.less [hunk 2]
```diff
@@ -1,42 +1,54 @@
...
+#appContainer {
+  &.errorPageContainer {
+    overflow: auto;
+    height: 100%;
+
+    .errorContent {
+      display: flex;
+      width: 40vw;
+      max-width: 600px;
+      margin: 20vh auto;
```
Comment (luixxiul) [inline less/about/error.less]: This was ``margin: 20vh auto 0;`` at first.

**Comment by diracdeltas** [comment]
looks good, thanks!

**Comment by luixxiul** [comment]
I found that ``actionButton`` should be applied to the "retry the url" button instead of ``wideButton``. Pushing a PR to fix that.

**Pull request merged**